omnibus/config/software/datadog-agent-integrations.rb

```diff
@@ -40,13 +48,51 @@
   # Copy the checks and generate the global requirements file
   block do
     all_reqs_file = File.open("#{project_dir}/check_requirements.txt", 'w+')
+    # Manually add "core" dependencies that are not listed in the checks requirements
+    # FIX THIS these dependencies have to be grabbed from somewhere
+    all_reqs_file.puts "requests==2.11.1 --hash=sha256:545c4855cd9d7c12671444326337013766f4eea6068c3f0307fb2dc2696d580e"
+    all_reqs_file.puts "pympler==0.5 --hash=sha256:7d16c4285f01dcc647f69fb6ed4635788abc7a7cb7caa0065d763f4ce3d21c0f"
+    all_reqs_file.puts "wheel==0.30.0 --hash=sha256:e721e53864f084f956f40f96124a74da0631ac13fbbd1ba99e8e2b5e9cafdf64"\
+        " --hash=sha256:9515fe0a94e823fd90b08d22de45d7bde57c90edce705b22f5e1ecf7e1b653c8"
+
+    all_reqs_file.close
+
+    # Install all the requirements
+    if windows?
+      pip_args = "install  -r #{project_dir}/check_requirements.txt"
+      command "#{windows_safe_path(install_dir)}\\embedded\\scripts\\pip.exe #{pip_args}"
```

Comment:
why not use the `pip` DSL method on windows too?

Reply:
It failed for me because it wasn't using the embedded pip, but what I presume was the "system" pip. We could further explore, Derek might have better knowledge of the windows omnibus pips.

Reply:
Added a card to track and verify this. I will try to do it soon if I get the chance, but otherwise the card will keep us honest.